roles/kubernetes/kubeadm/defaults/main.yml

```diff
@@ -10,3 +10,9 @@ kube_override_hostname: >-
   {%- else -%}
   {{ inventory_hostname }}
   {%- endif -%}
+
+# kubeadm patches path
```

Comment:
This part should have not been added to the defaults since you check for `kubeadm_patches is defined` which will always result in trying to apply patches even on default configurations where there are no patches.

In my own environment this broke down when trying to copy `/root/patches` since I store my inventory in `/root/inventroy.ini`. How was this not caught by CI?

/cc @floryut @oomichi

Reply:
Indeed my bad, but this was not caught by the CI as this PR also add a `patches` to the sample inventory which is used by the CI